src/View/OpenPatientWindow.py

```diff
@@ -324,9 +325,24 @@ def tree_item_clicked(self, item, _):
             header = "Only one study can be opened."
             self.open_patient_window_confirm_button.setDisabled(True)
 
+        # If RTSTRUCT exists but not selected, save existing RTSTRUCT file path
+        self.existing_rtss_path = self.get_existing_rtss_path(selected_series_types)
```

Comment:
```suggestion
        self.existing_rtss_path = \
            self.get_existing_rtss_path(selected_series_types)
```

PEP 8 - line too long (also lines 339, 342, 358).